### PR TITLE
Add count of entries (unfiltered). Fixes #2129.

### LINKED ISSUE #2129: Publish filtering option misses the count of entries
The new publish filtering is cool, but it doesn't display the count of entries. It would be useful to know

* how many entries are there if no filters are active
* how many entries have been found with the current filter(s)

### PR DESCRIPTION
**Pull request by michael-e**

### CODE REVIEW

File: symphony/content/content.publish.php [hunk 1]
```diff
@@ -495,7 +495,10 @@ public function __viewIndex()
 
         // Flag filtering
         if (isset($_REQUEST['filter'])) {
-            $filter_stats = new XMLElement('p', '<span>– ' . __('filtered (%d of %d)', array($entries['total-entries'], EntryManager::fetchCount($section_id))) . '</span>', array('class' => 'inactive'));
+            $filter_stats = new XMLElement('p', '<span>– ' . __('%d of %d entries (filtered)', array($entries['total-entries'], EntryManager::fetchCount($section_id))) . '</span>', array('class' => 'inactive'));
+            $this->Breadcrumbs->appendChild($filter_stats);
```
Comment (nilshoerrmann) [inline symphony/content/content.publish.php]: Can't we just have this line once after the condition? We'll always be adding the count, aren't we?

Comment (michael-e) [inline symphony/content/content.publish.php]: You mean line 499? I just thought that öater one might add more conditions, which theoroetically might not append `$filter_stats` (but something different, or nothing). Maybe I over-thought this?.

Comment (nilshoerrmann) [inline symphony/content/content.publish.php]: Haha, you just voted for always having a count and now you are already planning the pull-out :grinning:
I don't mind, I just noticed …

Comment (michael-e) [inline symphony/content/content.publish.php]: You are right, again I optimized for a "future scenario" that doesn't exist. I will add the simplification to the pull request.

Comment (michael-e) [inline symphony/content/content.publish.php]: https://github.com/michael-e/symphony-2/commit/0a084d774de18c06a3b3dee2314a1f5fc0d6d032

**Comment by nilshoerrmann** [comment]
Danke, Michael!

**Comment by nitriques** [comment]
Danke, Michael!

**Comment by michael-e** [comment]
No prob!

**Pull request merged**